packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -60,22 +62,27 @@ export enum FlushMode {
 	TurnBased,
 }
 
+/**
+ * @remarks Note: this API is experimental and not ready for production use.
```

Comment:
```suggestion
 * @remarks Note: this API is experimental and not be ready for production use.
```

Reply:
~Our docs should be in pirate-speak~ You're right, I'm not sure how exactly I misread this 😅 . Closing.